{"name": "test_1[2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 > 2", "trace": "data = 2\n\n    @pytest.mark.parametrize('data',[1,2,3])\n    def test_1(data):\n        case_data.case_test_record_list.append(\n            {'case_no': 'F0001', 'case_name': 'test01', 'case_story': '测试场景', 'exp_res': '预期结果', 'act_res': '实际结果'})\n>       assert data > 2\nE       assert 2 > 2\n\nD:\\autotest\\cases\\system\\AF\\8040\\test_param.py:17: AssertionError"}, "parameters": [{"name": "data", "value": "2"}], "start": 1614493648071, "stop": 1614493648072, "uuid": "b44d05d7-6119-461d-9976-2228876b24f7", "historyId": "868a813272257b42e08e8cea1a91d6fd", "testCaseId": "9b9c3028ba7355dc69be22b36d76be43", "fullName": "cases.system.AF.8040.test_param#test_1", "labels": [{"name": "parentSuite", "value": "cases.system.AF.8040"}, {"name": "suite", "value": "test_param"}, {"name": "host", "value": "DESKTOP-H5OTV7O"}, {"name": "thread", "value": "11816-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.system.AF.8040.test_param"}]}